vsintegration/src/unittests/Tests.LanguageService.Completion.fs

```diff
@@ -474,6 +474,75 @@ type AutoCompletionListTests() as this  =
                 "type B() = inherit A(System.String.)"
             ]
         AssertCtrlSpaceCompleteContains code "System.String." ["Empty"] ["Array"; "Collections"]
+    
+    [<Test>]
+    [<Category("Completion in object initializer")>]
+    member public this.``ObjectInitializer.CompletionForProperties``() =
+        let typeDef1 = 
+            [
+                "type A() = "
+                "   member val SettableProperty = 1 with get,set"
+                "   member val AnotherSettableProperty = 1 with get,set"
+                "   member val NonSettableProperty = 1"
+            ]
+        AssertCtrlSpaceCompleteContains (typeDef1 @ ["A((**))"]) "A((**)" ["SettableProperty"; "AnotherSettableProperty"] ["NonSettableProperty"] 
```

Comment:
It depends on whether parser was able to recover and understand that this was new expression or application. I have not tried to add all possible ways to guess if the cursor is in the context of object creation expression.